Formula/vvenc.rb

```diff
@@ -16,13 +16,9 @@ class Vvenc < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e1cbbed66a84f67a944f46dfa1b56d32e6c20b45c4daca6286d1faadea6325b4"
   end
 
+  depends_on "ccache" => :build
```

Comment:
This is weird. Why is it needed? It seems like this would evade our compiler shims, maybe.

Reply:
Good question. `ccache` won't actually do anything for us.

Reply:
it complains in the build log, i can remove it and give it another try.